include/test.h

```diff
@@ -0,0 +1,13 @@
+/*
+ *  * Copyright © 2017-2018 The Crust Firmware Authors.
+ *   * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ *    */
+
+#ifndef TEST
```

Comment:
This needs to follow the format of all the other header guards (should be `TEST_H`), otherwise it clashes with the macro used elsewhere.

Reply:
done